src/org/opendatakit/briefcase/ui/reused/UiLinkedFieldValidator.java

```diff
@@ -0,0 +1,47 @@
+package org.opendatakit.briefcase.ui.reused;
+
+import static org.opendatakit.briefcase.ui.reused.UiFieldValidator.COLOR_NOT_VALID;
+import static org.opendatakit.briefcase.ui.reused.UiFieldValidator.COLOR_VALID;
+
+public class UiLinkedFieldValidator {
```

Comment:
A doc comment here might help the reader.

Reply:
I was hoping that the tests would shed some light. I'll reference them in a doc block